modules/tests/src-ce2/UUIDGen.scala

```diff
@@ -19,6 +19,7 @@ package cats.effect.std
 import java.util.UUID
 import cats.effect._
 
+// Shim for https://github.com/typelevel/cats-effect/pull/2945 on CE2
```

Comment:
this replaces a note from Dependencies

Reply:
we've dropped support for CE2 haven't we ?

Reply:
oh yeah I think we did